charts/linkerd2/values.yaml

```diff
@@ -4,74 +4,73 @@
 
 # Values that are passed along to sub-charts
 global:
-  clusterDomain: &cluster_domain cluster.local
-  imagePullPolicy: &image_pull_policy IfNotPresent
-
-  # control plane version. See Proxy section for proxy version
-  linkerdVersion: &linkerd_version edge-19.12.3
-
-  namespace: linkerd
-
-  identityTrustAnchorsPEM: |
-
-  identityTrustDomain: *cluster_domain
-
-  # proxy configuration
-  proxy:
-    enableExternalProfiles: false
-    image:
-      name: gcr.io/linkerd-io/proxy
-      pullPolicy: *image_pull_policy
-      version: *linkerd_version
-    logLevel: warn,linkerd2_proxy=info
-    ports:
-      admin: 4191
-      control: 4190
-      inbound: 4143
-      outbound: 4140
-    resources:
-      cpu:
-        limit: ""
-        request: ""
-      memory:
-        limit: ""
-        request: ""
-    trace:
-      collectorSvcAddr: ""
-      collectorSvcAccount: default
-    uid: 2102
-    # If set, the proxy's pre-stop hook will postpone the Kubernetes's SIGTERM signal
-    # and wait for this duration before letting the proxy process the SIGTERM signal.
-    # See https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#container-hooks
-    # for more info on container lifecycle hooks.
-    waitBeforeExitSeconds: 0
-
-  # proxy-init configuration
-  proxyInit:
-    ignoreInboundPorts: ""
-    ignoreOutboundPorts: ""
-    image:
-      name: gcr.io/linkerd-io/proxy-init
-      pullPolicy: *image_pull_policy
-      version: v1.1.0
-    resources:
-      cpu:
-        limit: 100m
-        request: 10m
-      memory:
-        limit: 50Mi
-        request: 10Mi
-
-  # control plane annotations - do not edit
-  createdByAnnotation: linkerd.io/created-by
-  proxyInjectAnnotation: linkerd.io/inject
-  proxyInjectDisabled: disabled
-
-  # control plane labels - do not edit
-  controllerComponentLabel: linkerd.io/control-plane-component
-  controllerNamespaceLabel: linkerd.io/control-plane-ns
-  linkerdNamespaceLabel: linkerd.io/is-control-plane
-
+    clusterDomain: &cluster_domain cluster.local
+    imagePullPolicy: &image_pull_policy IfNotPresent
+
+    # control plane version. See Proxy section for proxy version
+    linkerdVersion: &linkerd_version edge-20.1.8
```

Comment:
AFACIT, this is the only needed change, right? Mind reverting the indentation changes, as we still have some other ongoing Helm PRs? Thanks.

Reply:
agh, sorry about that — this is, once again, VS Code's fault

Reply:
fixed, my bad!